environment

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
I don't think we should be including a druid environment file in the root of the application since the Druid environment setup is specific to each user's needs. I would be ok adding this into a `sample` directory though. What do you think?